### PR TITLE
Addressing issue #732

### LINKED ISSUE #732: (easy) - Fix loader API link
1. https://webpack.js.org/development/how-to-write-a-loader/ 
2. ` [[loader API | loaders]]` should point to `api/loaders/`.

### PR DESCRIPTION
**Pull request by rouzbeh84**
fixes #732 as well as updating [content/development/how-to-write-a-plugin.md](content/development/how-to-write-a-plugin.md) to change `instanceable` to `instantiated`

### CODE REVIEW

File: content/development/how-to-write-a-loader.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ This function is called when a resource should be transformed by this loader.
 
 In the simple case, when only a single loader is applied to the resource, the loader is called with one parameter: the content of the resource file as string.
 
-The loader can access the [[loader API | loaders]] on the `this` context in the function.
+The loader can access the [[loader API | loaders]](api/loaders/) on the `this` context in the function.
```
Comment (simon04) [inline content/development/how-to-write-a-loader.md]: It should be `[loader API](api/loaders/)`

Comment (rouzbeh84) [inline content/development/how-to-write-a-loader.md]: I thought so but I'd seen this double brackets elsewhere and thought it was specific to something else. Thanks for the clarification! Will update it now 👍

Comment (rouzbeh84) [inline content/development/how-to-write-a-loader.md]: or should I say 'i swear i've seen this elsewhere' haha

File: content/development/how-to-write-a-loader.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ This function is called when a resource should be transformed by this loader.
 
 In the simple case, when only a single loader is applied to the resource, the loader is called with one parameter: the content of the resource file as string.
 
-The loader can access the [[loader API | loaders]](api/loaders/) on the `this` context in the function.
+The loader can access the [loader API | loaders](api/loaders/) on the `this` context in the function.
```
Comment (simon04) [inline content/development/how-to-write-a-loader.md]: Not done yet: remove the ` | loaders` to end with `[loader API](api/loaders/)`. If you don't believe me, use the GitHub preview of the (changed) Markdown file.

**Comment by rouzbeh84** [comment]
oh my bad @simon04! I thought I was agreeing with you but that it was done on purpose as it's in other places. I know this may fall out of scope to the issue now but I (hopefully)resolved the same issue elsewhere now too :).

**Comment by bebraw** [comment]
Can you rebase? Good to go after that.

**Comment by rouzbeh84** [comment]
@bebraw sorry! i suck at this. tried to do it a few times but so much had happened since the history got rather mucked up for me even in GitKraken :(

**Comment by bebraw** [comment]
@rouzbeh84 No worries. 👍

**Pull request merged**